apps/archives/views.py

```diff
@@ -24,6 +24,7 @@
     'three_man_months',
     'women_in_symbols',
     'whirlwind',
+    'geo_timeline',
```

Comment:
Make sure this does not overwrite other views

Reply:
What do you mean? I don't think I understand.